fuel-client/Cargo.toml

```diff
@@ -21,6 +21,7 @@ path = "src/main.rs"
 
 [dependencies]
 chrono = { version = "0.4", features = ["serde"] }
+clap = { version = "3.1.2", features = ["derive"] }
```

Comment:
Any reason to not use major/minor as in the other deps?

Reply:
Agh sorry, good spot @vlopes11. I'll update these now.